app/serializers/comment_serializer.rb

```diff
@@ -2,10 +2,10 @@ class CommentSerializer < ActiveModel::Serializer
   embed :ids, include: true
   attributes :id, :body, :mentioned_usernames, :created_at, :updated_at, :parent_id, :parent_author_name, :versions_count
 
-  has_one :author, serializer: UserSerializer, root: 'users'
+  has_one :author, serializer: UserSerializer, root: :users
```

Comment:
I was running into a thing where AMS was serializing out a 'users' key and a :users  key, which b0rked things pretty badly. Normalized on symbols because that's what AMS defaults to.